src/components/Watch.svelte

```diff
@@ -13,33 +13,34 @@
     chatZoom,
     showCaption,
     chatSplit,
+    displayMode
   } from '../js/store.js';
   import {
     paramsVideoId,
     VideoSide,
     ChatSplit,
     paramsContinuation,
     paramsIsVOD,
-    paramsEmbedded
+    DisplayMode
   } from '../js/constants.js';
   import ChatEmbed from './ChatEmbed.svelte';
   import Popout from './Popout.svelte';
   import Captions from './Captions.svelte';
 
   document.title = 'LiveTL';
-  let isResizing = false;
+  import { isResizing } from '../js/store.js';
   let chatElem, vidElem, ltlElem;
   const resizable = (selector, info) => {
     j(typeof selector == 'string' ? document.querySelector(selector) : selector).resizable(info);
   };
   const convertPxAndPercent = () => {
     [
       [chatElem, $chatSplit == ChatSplit.VERTICAL ? 'width' : 'height', chatSize],
-      [paramsEmbedded ? null : vidElem, $videoSide == VideoSide.TOP ? 'height' : 'width', videoPanelSize]
+      [$displayMode === DisplayMode.FULLPAGE ? vidElem : null, $videoSide == VideoSide.TOP ? 'height' : 'width', videoPanelSize]
```

Comment:
This is kinda cluttered. You could move the ``videoSide`` ternary to a reactive statement since you use the same expression on line ``113``.